librad/src/git/include.rs

```diff
@@ -149,22 +147,25 @@ impl<Path> Include<Path> {
     }
 
     /// Generate an include file by giving it a `RadUrn` for a project and the
-    /// tracked `User`/`PeerId` pairs for that project.
+    /// tracked User handle/`PeerId` pairs for that project.
     ///
     /// The tracked users are expected to be retrieved by talking to the
     /// [`crate::git::storage::Storage`].
     #[tracing::instrument(level = "debug")]
-    pub fn from_tracked_users<I>(path: Path, local_url: LocalUrl, tracked: I) -> Result<Self, Error>
+    pub fn from_tracked_users<R, I>(
+        path: Path,
+        local_url: LocalUrl,
+        tracked: I,
+    ) -> Result<Self, Error>
```

Comment:
```suggestion
    ) -> Self
```

Reply:
Thanks, lost this while translating over from `maint` :sweat_smile: